### ISSUE TITLE
Map Solver Switching tabs on/off

### ISSUE THREAD

**Issue by Babalela01** [tsitu/MH-Tools#16]
The extra clicks seems to annoy some people and it looks strange on large screens, but I do not think we want completely separate tabbed/non-tabbed versions. 

I think this can be accomplished by simply removing the boostrap classes with JS when a checkbox is changed, or, alternatively, using CSS media queries.

**Comment by Babalela01** [comment]
This might however require replacing some of the tables with proper bootstrap layouts (Since bootstrap is included now anyway, it might as well be done).

**Comment by sguermond** [comment]
Yeah, I can probably do that easily. Tab view toggle at the top, and stack all the tables in the combined view. I won't be able to that until Monday, but it should be quick.

**Comment by tsitu** [comment]
What's the status on this? I temporarily reverted it back because many desktop users were complaining.

**Comment by sguermond** [comment]
I started working on this and got frustrated because I didn't have a solid idea of what I wanted to do with it. Getting back into it because I would really like to have a nice UI that is easy to use :) 

Basically my main pain points with the current layout are:
- a long location list pushes the mouse list super far down
- a mouse list with many locations makes the page very wide
- it's very annoying to use on mobile because the page is so big/the text is so small

I recognize that the tabbed layout also has annoying points, so it's not the right solution either.

Based on this and the user feedback you suggested, current requirements are:
- must be all on one page
- must be reasonable to navigate on mobile
- must accommodate different screen sizes

What do you think of a three-column layout somewhat like [this quick mockup](http://bit.ly/2iGrUTd)?
![screen shot 2017-01-03 at 8 50 12 pm](https://cloud.githubusercontent.com/assets/7174808/21631987/53c07a16-d1f6-11e6-8675-58fb7c68895c.png)

Changes from current layout:
- move the "invalid" alert to the top of the form
- location list is the first table
- mouse list (resized so it's a constant width like in my tabbed layout) is the second table

**Comment by tsitu** [comment]
I like this 3-column layout. Instructions should probably run across the full width on top. Also, will the relocated "invalid" alert be directly above the input textarea? If so, will it push the textarea down upon resizing?

**Comment by Babalela01** [comment]
Sophie also suggested grouping cells for the same location  (eg. Elub with shell and Gouda) when sorting by location. The feasibility and neccesity depends on how this layout comes out.
